digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1952,4 +1952,9 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
         }
     )
   }
+
+  delete("/unknownSpeedLimit/delete"){
+    val unknownSpeedLimitIds = parsedBody.extractOpt[Set[Int]]
```

Comment:
Ids normally use a Long type
```suggestion
    val unknownSpeedLimitIds = parsedBody.extractOpt[Set[Long]]
```